srcpkgs/XyGrib/template

```diff
@@ -0,0 +1,33 @@
+# Template file for 'XyGrib'
+pkgname=XyGrib
+version=1.0.1
+revision=1
+wrksrc="${pkgname}-${version}-src"
+patch_args="-Np1"
+build_style=gnu-makefile
+hostmakedepends="qt5-devel"
+makedepends="jasper-devel libnova-devel nettle-devel proj-devel qt5-devel MesaLib-devel librsvg-devel qwt-devel"
+short_desc="Weather data visulization"
+maintainer="Andrew Benson <abenson+void@gmail.com>"
+license="GPL-3.0-or-later"
+homepage="https://opengribs.org/"
+distfiles="https://github.com/opengribs/XyGrib/releases/download/v${version}/XyGrib-${version}-src.tgz"
+checksum=2309505161577a6bf4787b8b9e7ead4ca9dfdb05b3e655b4583d69bdb9f2be9e
+
+if [ -n $XBPS_CROSS ]; then
```

Comment:
You probably wanted to write `if [ "$CROSS_BUILD" ]; then` I'm not aware of any wariable called `XBPS_CROSS`.

Reply:
What's weird is that still fixed the build

Reply:
Yep because you had like `if TRUE` in there and therefore it was added always and setting LINK manually for normal build also obviously shouldn't break anything.